BackofficeBundle/Resources/public/coffee/groupTree/NodeTreeElementView.coffee

```diff
@@ -15,8 +15,15 @@ NodeTreeElementView = OrchestraView.extend(
     nodeGroupRoles = @options.group.model_roles.filter (element) ->
       element.model_id == nodeId
     formCollectionViewClass = appConfigurationView.getConfiguration('group_tab_node_tree_form', 'editEntityTab')
+    roles = @options.roles.roles
+
+    if nodeId == 'root'
+      # A root node can't be deleted
+      roles = @options.roles.roles.filter (element) ->
+        element.name != "ROLE_ACCESS_DELETE_NODE"
```

Comment:
We should think about a js file containing this constant: the risk is to put it on different location, and the day when it will be changed on the server side, the client side will be hard to maintain :/